bin/gem2rpm

```diff
@@ -45,6 +60,11 @@ end
 srpmdir = nil
 specfile = nil
 if options[:srpm]
+  unless File.exist?(gemfile)
```

Comment:
Under what conditions is this condition triggered? Wouldn't this fail earlier when "gemfile" would not exist?